bounties/level-2/stellar-accounts-as-a-service.md

```diff
@@ -0,0 +1,109 @@
+# Stellar accounts as a service
+
+## Award
+| Hunter | Reviewer | Type
+| :- | :- | :-
+| 2000 XLM | 800 XLM | Continuous
+
+📜&nbsp; View [existing submissions](https://github.com/tyvdh/stellar-quest-bounties/issues?q=is%3Aissue+stellar-accounts-as-a-service+) for this bounty. \
+🔵&nbsp; Start [hunting](https://github.com/tyvdh/stellar-quest-bounties/issues/new?assignees=&labels=&template=begin-the-hunt.yml&title=%F0%9F%94%B5+%60stellar-accounts-as-a-service.md%60) this bounty.
+
+## Description
+
+### What is this task?
+
+Each Stellar account needs to hold 1 XLM to be valid. This can be costly when companies like exchanges want to build
+on Stellar.
+
+Currently, exchanges assign each of their users a unique id that should be put in the memo field of a transaction when depositing.
+This approach is error-prone. Users can forget to add the memo, or they can input the wrong memo.
+
+To solve this issue [muxed accounts](https://github.com/stellar/stellar-protocol/blob/master/core/cap-0027.md) were added to Stellar. Muxed accounts are essentially a way to combine Stellar addresses with integer identifiers.
+Payments sent to a muxed account are actually sent to the account used to create the muxed account. However, the muxed account identifier is also added to the transaction.
+
+For this bounty the task is to create an API for a custodial wallet service using muxed accounts.
+A simple database should be used to keep track of the registered users, their muxed account addresses and their balances.
+
+The api should have the following endpoints:
+```
```

Comment:
Would you consider describingt the EPs in [OAS](https://swagger.io/specification/)?

Reply:
I'm not sure, I do think it's a good Idea but I think as it is right now is easier to understand than OpenApi.  Though I was not familiar with the OpenApi specification.

Reply:
I'll change the format to OpenApi [check](https://github.com/tyvdh/stellar-quest-bounties/pull/86#discussion_r684328110)